core/container/src/main/java/org/wildfly/swarm/container/runtime/ConfigurableManager.java

```diff
@@ -167,15 +169,27 @@ public ConfigurableManager(StageConfig stageConfig) {
         };
     }
 
+    public void rescan() throws IllegalAccessException, NoSuchMethodException, InvocationTargetException {
+        for (Object each : this.deferred) {
```

Comment:
Off topic: I feel the need to remind everyone that `Iterable` has a `forEach` method, so calling `stream` would be just a waste of code and time! :-)